src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -33,6 +33,25 @@ public static void Ctor_CharSpan(char[] valueArray, int startIndex, int length,
         }
 
         [Theory]
+        [InlineData("")]
```

Comment:
Should we include a test that verifies `Assert.Same(string.Empty, result)` when length is 0? Maybe also verify the action is not called?

Reply:
Fixed